webapp/main/basePaths.ts

```diff
@@ -114,3 +117,6 @@ export const getAssessmentPrintLink = (countryIso: any, assessmentType: any, onl
 export const odp = _generate(PARAMS.countryIso, PARAMS.assessmentType, FRAGMENTS.odp, PARAMS.tab)
 export const getOdpLink = (countryIso: any, assessmentType: any, sectionName: any, odpId?: any) =>
   _generate(countryIso, assessmentType, FRAGMENTS.odp, sectionName, odpId)
+
+// ==== Geo
+export const geo = '/geo'
```

Comment:
Dont change files under webapp/ unless absolutely necessary

Reply:
webapp is deprecated. or @sorja are we still using the paths from here?
in that case, for now we add the geo path here, and we'll move this later on.

@xbexbex please, make the geo a function. easier to add parameter if needed and consistent with other paths

Reply:
I just removed the path from the webapp for now